src/components/WorkshopPage/Workshop.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import ReactPlayer from 'react-player/youtube';
+import Container from '@material-ui/core/Container';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import useTheme from '@material-ui/core/styles/useTheme';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+
+
+const workshops = [
+	{
+		type: 'Web Development',
+		elements: [
+			{
+				title: 'Intro to HTML',
+				author: 'Eric',
+				description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit,',
+				url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO'
+			},
+			{
+				title: 'Intro to CSS',
+				author: 'Chandra',
+				description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit,',
+				url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO'
+			},
+			{
+				title: 'Intro to HTML',
+				author: 'Rick Astley',
+				description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit,',
+				url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO'
+			}
+		]
+	},
+	{
+		type: 'Mobile Development',
+		elements: [
+			{
+				title: 'Reactive Native',
+				author: 'Jakob',
+				description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit,',
+				url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO'
+			},
+			{
+				title: 'Android Studio',
+				author: 'Nareh',
+				description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit,',
+				url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO'
+			}
+		]
+	}
+];
+
+function Workshop() {
+	const theme = useTheme();
+	const isSmall = useMediaQuery(theme.breakpoints.down('sm'));
+
+	const workshopCards = workshops.map(item =>
+		<Grid key={item.type} style={{ paddingBottom: theme.spacing(isSmall ? 4 : 8) }}>
+			<Typography style={{ fontWeight: theme.typography.fontWeightMedium,
+				paddingBottom: theme.spacing(isSmall ? 4 : 8) }}
+			variant='h3' component='h2'>{item.type}
+			</Typography>
+			<Grid container spacing={8} justify='left'>
+				{item.elements.map(element => {
+					return (
+						<Grid key={element.title} item xs={12} sm={8} md={6}
```

Comment:
Currently, the display on `sm` is a bit suboptimal:

<img src="https://user-images.githubusercontent.com/1538624/106250817-25d2b080-61e2-11eb-9d9a-0d17f3b2f8cd.png" height="500">

Consider doing one of these two things:

1. Make the video take up the whole screen width:

   <img src="https://user-images.githubusercontent.com/1538624/106250929-4d297d80-61e2-11eb-996d-2a057a09e4eb.png" height="500">

2. Center all elements:

   <img src="https://user-images.githubusercontent.com/1538624/106251014-721df080-61e2-11eb-98a5-a1062daa2760.png" height="500">

Up to you which one you choose, but note that the [gallery page](https://hoth.uclaacm.com/gallery/) chose the latter approach.